raspberry_pi/voc_co2/CCS811_logger.py

```diff
@@ -0,0 +1,36 @@
+from time import sleep, time
+from Adafruit_CCS811 import Adafruit_CCS811
+import csv
+import os.path 
+
+def write2csv(data):
+	dataCSV = open('CCS811_log.csv', 'a')
+	log = csv.writer(dataCSV)
+	log.writerow(data)
+	dataCSV.close()
+
+ccs =  Adafruit_CCS811()
+
+while not ccs.available():
+	pass
+temp = ccs.calculateTemperature()
+ccs.tempOffset = temp - 25.0
```

Comment:
Where does this -25 comes from? If you're correcting a value because of some bias in the sensor, do create a function and provide a reference. It looks like a magic number to me.

Reply:
that section comes from the adafruit example.py

https://github.com/adafruit/Adafruit_CCS811_python/blob/3e7a86ccd84268cca0d37a7784013b815306688c/examples/CCS811_example.py#L8-L11

Reply:
@jygh98 but do you know what it means? Do they say anywhere else?

Reply:
I was able to find this comment from the [Adafruit_CCS811 library](https://github.com/adafruit/Adafruit_CCS811_python/blob/master/Adafruit_CCS811/Adafruit_CCS811.py#L153-L159).  From what it looks like, -25 is used somehow to determine the ambient temperature.  When using the example.py it didnt mention anything about needing to change the default -25 offset value.  

>
		''' Temperature is stored as an unsigned 16 bits integer in 1/512
		degrees there is an offset: 0 maps to -25C. The default value is
		25C = 0x64, 0x00. As an example 23.5% temperature would be
		0x61, 0x00.
		The internal algorithm uses these values (or default values if
		not set by the application) to compensate for changes in
		relative humidity and ambient temperature.'''